ansible/roles/debops.resources/defaults/main.yml

```diff
@@ -101,14 +101,14 @@ resources__templates: [ '{{ resources__src + "templates/by-group/all" }}' ]
                                                                    # ]]]
 # .. envvar:: resources__group_templates [[[
 #
-# List of paths containing the directories of all the groups the current host is in, based on the content of `group_names`.
+# List of paths containing the directories of all the groups the current host is in, based on the content of ``group_names``.
 # See `Ansible - Playbooks Varibles <https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#accessing-information-about-other-hosts-with-magic-variables>`_.
 #
-# For example if the host `debian1` is member of `group-name1` and `group-name2`
-# `debops` will then search all template files inside the directories placed here: :file:`ansible/resources/template/by-group/`.
-# Resulting in: `[ "ansible/resources/template/by-group/group-name1", "ansible/resources/template/by-group/group-name3" ]`.
+# For example if the host ``debian1`` is member of ``group-name1`` and ``group-name2``
+# `debops.resources` will then search all template files inside the directories placed here: :file:`ansible/resources/templates/by-group/`.
+# Resulting in: ``[ "ansible/resources/template/by-group/group-name1", "ansible/resources/template/by-group/group-name3" ]``.
```

Comment:
The `template` directory here is also misspelled, and you have `group-name2` instead of `group-name3`. Just trying to be consistent. :-)

Reply:
This fixes are taking longer than writing the code itself :D